src/App.tsx

```diff
@@ -1,61 +1,144 @@
 import './App.scss';
+import { useState } from 'react';
+import { Todo } from './types/Todo';
+import { User } from './types/User';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+const isExist = (todo: Todo) => {
```

Comment:
```suggestion
const isUserWithTodo = (todo: Todo) => {
```

Reply:
I think it would be even better to call it similar to the function in the checklist, like userWithTodo